### ISSUE TITLE
Satellite gifs do not expand or rotate

### ISSUE THREAD

**Issue by midair55** [VA3HDL/hamdashboard#11]
Pablo -

Most likely not worth a "fix" but should be noted....

Neither of the images below expand to full screen when double clicked.   A small image appears in the upper left corner of the page and double clicking on it returns to the full dashboard screen.  The images also do not rotate like the others when combined in a single panel.

[
    "SATELLITE CAN",
    "https://cdn.star.nesdis.noaa.gov/GOES16/GLM/SECTOR/can/EXTENT3/GOES16-CAN-EXTENT3-1125x560.gif",
  ],
  [
    "SATELLITE CGL",
    "https://cdn.star.nesdis.noaa.gov/GOES16/GLM/SECTOR/cgl/EXTENT3/GOES16-CGL-EXTENT3-600x600.gif",
  ],

**Comment by midair55** [comment]
Hmmmm....  it was consistently failing but now working much better;  expansion and rotation.  I get the following errors at times infrequently.   Internet speed is typically 400Kbps down.   Perhaps we can blame it on the NOAA servers.....

Failed to load image https://cdn.star.nesdis.noaa.gov/GOES16/GLM/SECTOR/se/EXTENT3/GOES16-SE-EXTENT3-600x600.gif?_=1717704894911
dash.html:444 Trying without caching prevention
dash.html:441 Failed to load image https://cdn.star.nesdis.noaa.gov/GOES16/GLM/SECTOR/se/EXTENT3/GOES16-SE-EXTENT3-600x600.gif?_=1717704908929
dash.html:444 Trying without caching prevention

**Comment by VA3HDL** [comment]
I'm attempting to download the image first with a random string at the end to prevent caching. If that fails, then the 2nd load attempt is without the ?_=nnnnnn part. If that fails, then it is something else wrong.

I just tested on my work computer running Microsoft Edge and I got the little square icon of the missing image once and I noticed the image was not completely loaded. So it could happen on slow connections. Is a big file 21 MB each time that shows on the screen (due to the caching prevention.) The other file is like 12 MB.

I will add options on the config to permit the caching or to use every N seconds. I don't like to generate excessive traffic downloading the exact same image hundred times per hour.

I'm glad is working better for you.
